shell.js

```diff
@@ -23,7 +23,7 @@ exports.pwd = common.wrap('pwd', _pwd);
 
 //@include ./src/ls
 var _ls = require('./src/ls');
-exports.ls = common.wrap('ls', _ls, {idx: 1});
+exports.ls = common.wrap('ls', _ls, {noGlob: true});
```

Comment:
`config.noGlob` is real, `options.noGlob` isn't. You're specifying `options.noGlob` here (just like how `idx` is really `options.index` [here](https://github.com/shelljs/shelljs/blob/master/src/common.js#L275))